feature-libs/quote/components/details/comment/quote-details-comment.component.html

```diff
@@ -1,20 +1,25 @@
 <ng-container *ngIf="quoteDetails$ | async as quoteDetails">
-  <div class="quote-comment-toggle" (click)="showComments = !showComments">
-    <cx-icon
-      aria-hidden="false"
-      [type]="showComments ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
+  <ng-container *ngIf="showComments(quoteDetails)">
+    <div
+      class="quote-comment-toggle"
+      (click)="expandComments = !expandComments"
```

Comment:
Please, define a method for toggling the comments area.

Reply:
@Uli-Tiger, the review is for putting out, maybe, a better suggestion. It is up to you to follow it or not.
I would understand better if I see something like `toggleComments` instead of this condition